azure-pipelines.yml

```diff
@@ -222,26 +223,30 @@ jobs:
       ci/run_verilator_pytest.sh
     displayName: 'Execute tests'
 
-- job: "riscv_compliance_tests"
-  displayName: "Execute RISC-V compliance tests"
-  pool: "Default"
-  dependsOn:
-    - top_earlgrey_verilator
-    - execute_verilated_tests
-  steps:
-  - template: 'ci/install-package-dependencies.yml'
-  - template: 'ci/download-artifacts-template.yml'
-  - bash: |
-      . util/build_consts.sh
-      export TARGET_SIM="$BIN_DIR/hw/top_earlgrey/Vtop_earlgrey_verilator"
-      export RISCV_DEVICE=rv32imc
-      export RISCV_TARGET=opentitan
-      export OT_TARGET=verilator
-      export OT_TOOLS="$TOOLCHAIN_PATH/bin"
-      cd sw/vendor/riscv_compliance
-      # Only running base interger variant for now, others will be added later.
-      make RISCV_ISA=rv32i
-    displayName: 'Execute tests'
+##  riscv compliance job is temporarily disabled pending some issues related to the
+##  makefile / meson conversion.  See issue #1181
+
+##- job: "riscv_compliance_tests"
+##  displayName: "Execute RISC-V compliance tests"
+##  pool:
+##    vmImage: "ubuntu-16.04"
+##  dependsOn:
+##    - top_earlgrey_verilator
+##    - execute_verilated_tests
+##  steps:
+##  - template: 'ci/install-package-dependencies.yml'
+##  - template: 'ci/download-artifacts-template.yml'
+##  - bash: |
+##      . util/build_consts.sh
+##      export TARGET_SIM="$BIN_DIR/hw/top_earlgrey/Vtop_earlgrey_verilator"
+##      export RISCV_DEVICE=rv32imc
+##      export RISCV_TARGET=opentitan
+##      export OT_TARGET=verilator
+##      export OT_TOOLS="$TOOLCHAIN_PATH/bin"
+##      cd sw/vendor/riscv_compliance
+##      # Only running base interger variant for now, others will be added later.
+##      make RISCV_ISA=rv32i
+##    displayName: 'Execute tests'
 
```

Comment:
You're better off adding `condition: False` to disable a job. It's a smaller diff and still makes the disabled job show up in CI.